.github/workflows/version_check.yml

```diff
@@ -0,0 +1,198 @@
+name: Check Turing.jl version consistency
+on:
+  push:
+    branches:
+      - master
+      - backport-*
+  pull_request:
+    branches:
+      - master
+      - backport-*
+  workflow_dispatch:
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    env:
+      # Determine whether the target branch is master (i.e. this is a push to
+      # master or a PR to master).
+      TARGET_IS_MASTER: ${{ (github.event_name == 'push' && github.ref_name == 'master') || (github.event_name == 'pull_request' && github.base_ref == 'master') }}
+      # Disable precompilation as it takes a long time and is not needed for this workflow
+      JULIA_PKG_PRECOMPILE_AUTO: 0
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Julia
+        uses: julia-actions/setup-julia@v2
+        
+      - name: Log GitHub context variables
+        run: |
+          echo github.event_name: ${{ github.event_name }}
+          echo github.ref_name: ${{ github.ref_name }}
+          echo github.base_ref: ${{ github.base_ref }}
+          echo TARGET_IS_MASTER: ${{ env.TARGET_IS_MASTER }}
+
+      - name: Check version consistency
+        shell: julia --color=yes --project=. {0}
+        continue-on-error: true
+        run: |
+          using Pkg
+          Pkg.activate(temp=true)
+          Pkg.add(["YAML", "TOML", "JSON", "HTTP"])
+          import YAML
+          import TOML
+          import JSON
+          import HTTP
+
+          PROJECT_TOML_PATH = "Project.toml"
+          QUARTO_YML_PATH = "_quarto.yml"
+          MANIFEST_TOML_PATH = "Manifest.toml"
+
+          function major_minor_match(vs...)
```

Comment:
Is this equivalent to `all(v[1] == v for v in vs)` except it ignores `patch`, `prerelease`, and `build`? Could it have a short docstring to say as much?

Reply:
I've changed the implementation to use all(). Do you think it still needs a docstring to explain what the code does?